File: Courzelo-backend-LMS/Courzelo/src/main/java/org/example/courzelo/dto/responses/UserEducationResponse.java
package org.example.courzelo.dto.responses;

import lombok.Builder;
import lombok.Data;
import org.example.courzelo.dto.responses.institution.SimplifiedClassRoomResponse;

import java.util.List;


@Data
@Builder
public class UserEducationResponse {
    private String institutionID;
    private String institutionName;
    private List<SimplifiedClassRoomResponse> classrooms;
    private String groupID;

}

File: courzelo-dashboard-LMS-main/src/app/shared/models/user/UserEducationResponse.ts
/*import {SimplifiedClassRoomResponse} from '../institution/SimplifiedClassRoomResponse';

export interface UserEducationResponse {
    institutionID?: string;
    institutionName?: string;
    classrooms?: SimplifiedClassRoomResponse[];
    groupID?: string;
}*/
import { SimplifiedClassRoomResponse } from '../institution/SimplifiedClassRoomResponse';

export interface InstitutionForUser {
  institutionID: string;
  institutionName: string;
  classrooms?: SimplifiedClassRoomResponse[];
}

export interface UserEducationResponse {
  institutionID?: string;
  institutionName?: string;
  classrooms?: SimplifiedClassRoomResponse[];
  groupID?: string;

  // Ajouté pour les utilisateurs multi-institution (ex: TEACHER)
  institutions?: InstitutionForUser[];
}